src/lib/methods/method84.ts

```diff
@@ -30,18 +30,24 @@ export default (number: string): Result => {
     return method51NominalAccount(paddedNumber);
   }
 
+  const significantNumber = paddedNumber.slice(4, 9);
```

Comment:
You have to use `.slice(4, 10)` since method06Core below requires the significant digits for calculation (which you indeed get with `4, 9`, but also the check digit which is the 10th digit).

Reply:
good catch, fixed